

include/asm/errno.h:44:1: warning: this is the location of the previous definition
In file included from fs/reiser4/sys_reiser4.c:22:
include/asm-generic/errno.h:95:1: warning: "EDQUOT" redefined
In file included from include/linux/errno.h:4,
                 from include/linux/quota.h:38,
                 from include/linux/fs.h:295,
                 from fs/reiser4/sys_reiser4.c:9:
include/asm/errno.h:43:1: warning: this is the location of the previous definition
In file included from fs/reiser4/sys_reiser4.c:22:
include/asm-generic/errno.h:97:1: warning: "ENOMEDIUM" redefined
In file included from include/linux/errno.h:4,
                 from include/linux/quota.h:38,
                 from include/linux/fs.h:295,
                 from fs/reiser4/sys_reiser4.c:9:
include/asm/errno.h:102:1: warning: this is the location of the previous definition
In file included from fs/reiser4/sys_reiser4.c:22:
include/asm-generic/errno.h:98:1: warning: "EMEDIUMTYPE" redefined
In file included from include/linux/errno.h:4,
                 from include/linux/quota.h:38,
                 from include/linux/fs.h:295,
                 from fs/reiser4/sys_reiser4.c:9:


Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-sparc64-akpm/fs/reiser4/sys_reiser4.c |    2 +-
 1 files changed, 1 insertion(+), 1 deletion(-)

diff -puN fs/reiser4/sys_reiser4.c~sys_reiser4-sparc64-build-fix fs/reiser4/sys_reiser4.c
--- 25-sparc64/fs/reiser4/sys_reiser4.c~sys_reiser4-sparc64-build-fix	2004-08-18 23:21:11.139635696 -0700
+++ 25-sparc64-akpm/fs/reiser4/sys_reiser4.c	2004-08-18 23:21:17.548661376 -0700
@@ -19,7 +19,7 @@
 #include <linux/module.h>
 #include <linux/writeback.h>
 #include <linux/backing-dev.h>
-#include <asm-generic/errno.h>
+#include <linux/errno.h>
 
 #if defined(CONFIG_REISER4_FS_SYSCALL)
 
_
